pkg/schema/fake_registry.go

```diff
@@ -94,33 +91,29 @@ const (
 	errorCodeSchemaNotFound  = 40403
 )
 
-// fakeRegistry is a simple fake registry meant to be used in tests. It stores
+// FakeRegistry is a simple fake registry meant to be used in tests. It stores
 // schemas in memory and supports only the basic functionality needed in our
 // tests and supported by our client.
-type fakeRegistry struct {
+type FakeRegistry struct {
```

Comment:
I wonder if `FakeRegistry` is a suitable name now that we don't use it solely as a fake in tests but also as a real in-memory registry.

Reply:
Good point, I'll rename it!